guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -1,10 +1,10 @@
 [id="upgrading-{project-context}-in-place-using-leapp_{context}"]
-= Upgrading {Project} to {EL} 8 In-Place Using Leapp
+= Upgrading {Project} or {SmartProxy} to {EL} 8 In-Place Using Leapp
```

Comment:
Please also adjust the anchor and file name accordingly.

Reply:
I didn't find any web link that would break after changing this anchor, only one xref. You can easily find affected xrefs by running by running `git grep 'xref:upgrading-{project-context}-in-place-using-leapp_'` from the `foreman-documentation` directory and edit the found files either manually or via sed. In this case, the xref is in `guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc`.

Reply:
I meant external links, like the ones I've posted multiple times on https://community.theforeman.org